packages/components/src/multi-select/multi-select.base.tsx

```diff
@@ -29,6 +31,9 @@ const defaultProps: Partial<MultiSelectBaseProps> = {
 	hiddenInputValuesDivider: ',',
 };
 
+const SELECT_ALL_VALUE = 'all';
+const NA_VALUE = '-1';
```

Comment:
N/A value is not fixed as `-1` as it can be `-2` or `N/A`, the option to include N/A is not necessary